### DISCUSSION TITLE
Help! Exception, Driver exited No config found, Please create one

### DISCUSSION THREAD

**Discussion by ahalves** [anilabhadatta/educative.io_scraper#38] [Q&A] [answered]
What's the problem here? Maybe I don't know how to make a config but I'm pretty sure I did, can you clarify how to do so?

![image](https://user-images.githubusercontent.com/127245078/224588131-df80c769-6527-4dd8-bef4-17ed77b00add.png)

**Comment by anilabhadatta** [comment]
https://github.com/anilabhadatta/educative.io_scraper#how-to-use-the-scraper
Read the steps correctly

**Comment by ahalves** [comment, marked as answer]
Didn't work, but I tried 2 more times and then it worked 🤣

**Reply by anilabhadatta** [reply]
@ahalves version 3 is released, do check it out.
